README.md

```diff
@@ -2,4 +2,31 @@
 
 Soroban DApps is a collection of demos and experiments showcasing the capabilities of Soroban smart contracts. Each folder within this repository represents a different DApp, offering a README file with detailed information about its purpose and usage. 
 
-Please note that the demos and experiments included in this repository are meant for educational and illustrative purposes. They can help developers get started with Soroban and serve as a foundation for building more sophisticated and customized applications.
\ No newline at end of file
+Also have examples of how to use *[Stellar Plus](#stellar-plus-library)* Library, an all-in-one Javascript library for building and interacting with the Stellar network. Stellar Plus bundles the main resources from the community into an easy-to-use set of tools and capabilities.
+
+Please note that the demos and experiments included in this repository are meant for educational and illustrative purposes. They can help developers get started with Soroban and serve as a foundation for building more sophisticated and customized applications.
+
+## Stellar Plus Library
+The stellar-plus library is a powerful tool for interacting with Stellar blockchain. For detailed documentation, refer to the [Stellar Plus Documentation](https://cheesecake-labs.gitbook.io/stellar-plus).
```

Comment:
What do you think about including the link to the docs in the paragraph above that already mentions stellar-plus and removing this part?